aleph/migration.py

```diff
@@ -34,7 +34,7 @@ def destroy_db():
     while len(tables):
         for table in tables:
             try:
-                table.drop(checkfirst=True)
+                table.drop(bind=db.engine, checkfirst=True)
```

Comment:
This isn’t strictly related to this PR, but I think it might have been overlooked during the SQLAlchemy 2 migration and it caused migrations to fail in my development environment.